docs/source/reference/numpysupported.rst

```diff
@@ -133,6 +133,96 @@ The following constructors are supported, only with a numeric input:
 * :class:`numpy.uintp`
 
 
+Modules
+=======
+
+.. _numpy-random:
+
+``random``
+----------
+
+Numba supports top-level functions from the
+`numpy.random <http://docs.scipy.org/doc/numpy/reference/routines.random.html>`_
+module, but does not allow you to create individual RandomState instances.
+The same algorithms are used as for :ref:`the standard
+random module <pysupported-random>` (and therefore the same notes apply),
+but with an independent internal state: seeding or drawing numbers from
+one generator won't affect the other.
+
+The following functions are supported, but only with scalar output: you can't
+pass a *size* argument.
+
+Initialization
+''''''''''''''
+
+* :func:`numpy.random.seed`: with an integer argument only
+
+Simple random data
+''''''''''''''''''
+
+* :func:`numpy.random.rand`: only without argument
+* :func:`numpy.random.randint`
+* :func:`numpy.random.randn`: only without argument
```

Comment:
This seems to work, but I couldn't find a test for it (everything else apart from `random_integers` I could see a test for).